stumpy/scraamp.py

```diff
@@ -325,7 +395,8 @@ class scraamp:
     update()
         Update the matrix profile and the matrix profile indices by computing
         additional new distances (limited by `percentage`) that make up the full
-        distance matrix.
+        distance matrix. Each output contains three columns that are corresponding
```

Comment:
"that correspond to the matrix profile, the left matrix profile, and the right matrix profile, respectively"